internal/scm/gitea/gitea.go

```diff
@@ -420,3 +425,62 @@ func (g *Gitea) ClosePullRequest(ctx context.Context, pullReq domain.PullRequest
 
 	return nil
 }
+
+// ForkRepository forks a repository. If newOwner is empty, fork on the logged in user
+func (g *Gitea) ForkRepository(ctx context.Context, repo domain.Repository, newOwner string) (domain.Repository, error) {
+	r := repo.(repository)
+
+	var forkTo string
+	if forkTo == "" {
```

Comment:
uh... is there any case where this condition would be false?

Reply:
Good catch! This should ofc be `newOwner`